README.md

```diff
@@ -1 +1,218 @@
-# token-issuer
\ No newline at end of file
+# ZGW Token Issuer
+
+| Key       | Value                                           |
+|-----------|-------------------------------------------------|
+| Version   | 1.0.0                                           |
+| Source    | https://github.com/VNG-Realisatie/token-issuer  |
+| Keywords  | ZGW, tooling                                    |
+| Related   | https://github.com/VNG-Realisatie/token-seeder  |
+
+
+## Introductie
+
+Om het zaakgericht werken in de Common Ground architectuuur te ondersteunen zijn de [API's voor Zaakgericht werken ontwikkeld](https://github.com/VNG-Realisatie/gemma-zaken). 
+Onderdeel van deze API's is een autorisatie module, de [Autorisaties API](https://github.com/VNG-Realisatie/autorisaties-api) welke tokens gebruikt. 
+Deze tokens dienen genereerd en ingericht te worden in de verschillende API's. 
+
+Ten behoeve van zaakgericht werken is er een nieuwe Tokentool ontwikkeld in fastapi. Deze dient als vervanging voor de bestaande token-tool
+om deze binnen VNGr te krijgen.
+
+De werking heeft de GUI achterwege gelaten en werkt nu als een pure restapi. Hierdoor is het aantal endpoints (en werking) sterk gedaald.
+
+
+## Swagger en Redoc
+
+Bij de token-issuer is om de werking te vereenvoudigen swagger en redoc documentatie toegevoegd. Deze worden op het standaardadres geserveerd.
+
+{base_address} dient vervangen te worden door de instelling van je `ingress` met een port-forward is het meestal `http://localhost:8000`
+
+
+#### swagger:
+
+{base_address}/docs
+
+#### redoc:
+
+{base_address}/redoc
+
+#### openapi
+
+{base_address}/api/v1/openapi.json
```

Comment:
Domme vraag misschien maar Is er geen openapi.yaml?

Reply:
Dat klopt en dat is een bewuste keuze van fastapi. Yaml betekent namelijk een extra dependency (zie discussie hier):

https://github.com/tiangolo/fastapi/issues/1140

voor de werking van redoc of swagger is er ook geen yaml nodig. We kunnen wel zoiets inbouwen met wat gegoochel door te zeggen bij opstarten dat er een endpoint moet zijn openapi.yaml die de json ophaalt en parsed naar yaml maar ik zie daar eerlijk gezegd de meerwaarde niet van in. 

Heeft dit te maken met de design regels die stellen dat je een yaml moet hebben?

Reply:
Voor zover ik weet is een yaml niet per se verplicht.  Als ik eens kijk naar https://stackoverflow.com/questions/46990638/what-is-the-practical-different-between-the-usage-of-json-and-yaml-in-swagger 
Zou mijn voorkeur uitgaan naar JSON. Yaml vind ik een draak van een formaat. Alleen al dat een spatie meer of minder het bestand ongeldig maakt, dat is zo 1831 ...
Dus wat mij betreft stappen we over op JSON voor de API specificaties

Reply:
https://publicatie.centrumvoorstandaarden.nl/api/adr/1.0/#api-51

zegt openapi.json en eventueel yaml. Dus dit is helemaal goed (voor zover het nodig is voor een tool).